CHANGELOG_PENDING.md

```diff
@@ -66,3 +68,4 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 ### BUG FIXES:
 
 - [consensus] [\#4895](https://github.com/tendermint/tendermint/pull/4895) Cache the address of the validator to reduce querying a remote KMS (@joe-bowman)
+- [rpc] \#4937 Return an error when `page` pagination param is 0 (@melekes)
```

Comment:
This should be listed as a breaking change too. Changes in the RPC API are more severe than changes in the Go API, since we guarantee backwards compatibility for it across patch releases.